docs/guides/embeds.md

```diff
@@ -0,0 +1,128 @@
+# Embeds
+
+Embeds in Discord are just **embedded content** that you can use to **organize the response of the bot**, with **title**, **image**, **footer**, and embed **color**. And as a benefit, it has a higher message content limit on its description.
```

Comment:
Embeds in Discord are just **embedded content messages** that you can use to **organize the response of the bot**, with **larger titles**, **images**, **footers**, and embed **colors**. As an added benefit, they have a higher message content limit on its description. Embeds can be used to construct clickable dialogs, for example